src/App.tsx

```diff
@@ -17,18 +58,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                setSelectStatus={setSelectStatus}
+              />
             </div>
 
-            <div className="block">
-              <Loader />
-              <TodoList />
-            </div>
+            {todos.length === 0
```

Comment:
Let's create a boolean useState `isLoading` and use it here. 
Initially, it will be `true`, after you get your todos from the server change it to `false`.

```suggestion
            {isLoading
```